### DISCUSSION TITLE
PPOCRLabel手动标记的文本框导出识别时会被忽略

### DISCUSSION THREAD

**Discussion by Gerres** [PaddlePaddle/PaddleOCR#12715] [Q&A]
在KIE和非KIE模式下，手动对图片打标注框，确认图片后，导出识别结果时crop_img为空，没有报错，文本路径也没有中文名
这是本身就不支持手动标注后导出识别还是BUG。。

**Comment by yanshuikafei** [comment]
可以修改生成的Label.txt里的"difficult": false

**Comment by SWHL** [comment]
因PPOCRLabel已经搬家了，后续有问题请移步[PPOCRLabel](https://github.com/PFCCLab/PPOCRLabel)提问哈！
此issue先行关闭了。
